dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -34,16 +34,56 @@ view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
 
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
+{% if SNMP_COMMUNITY is defined %}
+
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
+rocommunity {{ community }}
+rocommunity6 {{ community }}
+{% elif SNMP_COMMUNITY[community]['TYPE'] == 'RW' %}
+rwcommunity {{ community }}
+rwcommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_COMMUNITY is defined and snmp_rocommunities is defined %}
 {% for community in snmp_rocommunities %}
+{% if community not in SNMP_COMMUNITY %}
 rocommunity {{ community }}
 rocommunity6 {{ community }}
+{% endif %}
 {% endfor %}
-{% else %}
+{% elif community not in SNMP_COMMUNITY %}
```

Comment:
>community [](start = 8, length = 9)

`community` is in the scope of above `for/endfor` block?

Reply:
old comment not needed after update.